extensions/processing/README.md

```diff
@@ -43,22 +47,17 @@ This list is not exhaustive and can be extended with the processing level specif
-| L1B         | Level 1A data that have been processed to sensor units (not all instruments have Level 1B source data). | [Sentinel-2 L1B](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/product-types/level-1b) |
-| L1C         | Level 1B data that have been processed on a on the reference ellipsoid and represented in a uniform preselected cartographic presentation. | [Sentinel-2 L1C](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/product-types/level-1c) |
-| L1D         | Level 1C data that have been processed to generate a geocoded terrain corrected product which is fully calibrated through the use of a terrain model, then detected, geolocated on a DEM (Digital Elevation Model), and represented in a uniform pre-selected cartographic projection. | [KOMPSAT-5 L1D](https://directory.eoportal.org/web/eoportal/satellite-missions/k/kompsat-5) |
-| L1TP        | Radiometrically calibrated and orthorectified product using ground control points (GCPs) and digital elevation model (DEM) data to correct for relief displacement. | Landsat7/8 L1TP |
-| L1GT        | Radiometrically calibrated product with systematic geometric corrections applied using the spacecraft ephemeris data and DEM data to correct for relief displacement. | Landsat7/8 L1GT |
-| L1GS        | Radiometrically calibrated product with only systematic geometric corrections applied using the spacecraft ephemeris data. | Landsat7/8 L1GS |
 | L2          | Retrieved environmental variables (e.g., ocean wave height, soil-moisture, ice concentration) at the same resolution and location as the level 1 source data. A wide variety of sub-level products are possible (see below). | |
-| L2A         | Derived geophysical variables at the same resolution and location as Level 1 source data applying specific algorithm such as scene classification and atmospheric correction | [Sentinel-2 L2A](https://earth.esa.int/web/sentinel/technical-guides/sentinel-2-msi/level-2a-processing) |
-| L2SP        | Surface reflectance and surface temperature science products | [Landsat Level 2 science products](https://www.usgs.gov/core-science-systems/nli/landsat/landsat-collection-2-level-2-science-products) |
 | L3          | Data or retrieved environmental variables which have been spatiallyand/or temporally re-sampled (i.e., derived from level 1 or 2 products). Such re-sampling may include averaging and compositing.  A wide variety of sub-level products are possible (see below). | [ENVISAT Level-3](http://envisat.esa.int/level3/), [Sentinel-2 L3](https://s2gm.sentinel-hub.com/) |
 | L4          | Model output or results from analyses of lower level data (i.e.,variables that are not directly measured by the instruments, but are derived fromthese measurements) |
 
+## Implementations
+
+[DotNetStac](https://github.com/Terradue/DotNetStac) uses this extension specification for the processing extension plugin with accessor to the value specified of this extension.
```

Comment:
Depending on the outcomes in https://github.com/radiantearth/stac-spec/pull/920 we might ask you to (re)move this.

Reply:
Fine with me

Reply:
Though do you have catalogs produced with dotnetstac that use it? Even if they aren't public, you can describe it and say how large a catalog it is, or how many catalogs have been produced with DotNetStac. I think you could mention they are produced by DotNetStac and link to it, but the main implementation bullet point should be about the catalogs (also ok if none are produced yet / haven't yet been done with the latest changes, just add it in when they are).

Reply:
No public catalog so far. I will update as soon as we have more consistent examples. I remove the reference to DotNetStac to move forward with the merge. No problem with that.